src/diagnostics/LogPointsManager.ts

```diff
@@ -105,7 +108,13 @@ class DebugSessionManager {
     }
 
     public async removeLogpoint(logpoint: ILogpoint): Promise<void> {
-        await this._debugSession.customRequest("removeLogpoint", logpoint.id);
+        const result: IRemoveLogpointResponse = await this._debugSession.customRequest("removeLogpoint", logpoint.id);
+        if (result.error !== undefined && result.error !== null) {
+            const errorMessage = result.error.message === '' ?
+                'Cannot remove logpoint. Please refer to https://aka.ms/logpoints for more details.' :
+                `Cannot remove logpoint, the error is "${result.error.message}". Please refer to https://aka.ms/logpoints for more details.`;
+            throw new Error(errorMessage);
```

Comment:
The error dumping statements are removed in favor of throwing an exception. Because the underlying exception capturing logic will both showing a error prompt and logging a message to the output pane. It is done before sending out the telemetry.